theme/less/components/multiselect.less

```diff
@@ -0,0 +1,63 @@
+@import (less) "@conciergerie.dev/select-a11y/dist/select-a11y.css";
+.multiselect {
+    &[data-selected=true] .select-a11y .select-a11y-button {
+        color: @grey-500;
+    }
+    .select-a11y {
+        .fr-mt-1v;
+        padding: 0;
+        .select-a11y-button {
+            padding: 0.5rem 2.5rem 0.5rem 1rem;
+            background: none;
+            color: @text-mention-grey;
+            div {
+                overflow: hidden;
+                white-space: nowrap;
+                text-overflow: ellipsis;
+                display: block;
+            }
+            .select-a11y-button__icon {
+                display: none;
+            }
+        }
+        .select-a11y-button__clear::before {
+            &:extend(.fr-fi-close-circle-line::before);
+            &:extend([class^="fr-fi-"]::before);
+        }
+        .select-a11y__overlay {
```

Comment:
Can you reload your page after resizing your browser ?
For now, the component only calculate the offset based on viewport on boot and not on resize to avoid adding an event listener. We can add it too if you think it's necessary.

Reply:
Even when reloading, I have the same behavior in FF or chrome (I'm using the responsive mode with a 310*550 resolution).
But when navigating with my mobile directly, it seems to show nicely.
I think it fails when horizontal resolution is too low.